src/components/forms/validations.jsx

```diff
@@ -44,5 +45,5 @@ module.exports.validationHOCFactory = function (defaultValidationErrors) {
 };
 
 module.exports.defaultValidationHOC = module.exports.validationHOCFactory({
-    isDefaultRequiredValue: 'This field is required'
+    isDefaultRequiredValue: <intl.FormattedMessage id="teacherRegistration.validationRequired" />
```

Comment:
Actually looking again at where this is, it's already used in any component that has a required field, like the login form.
![image](https://cloud.githubusercontent.com/assets/1512021/20972994/da9c65ec-bc64-11e6-9125-421e93bcc69b.png)